snakes_battle/fruit.py

```diff
@@ -4,28 +4,62 @@ class FruitKind:
         "color": (255, 62, 97),
         "image": "snakes_battle\\images\\fruits\\strawberry.png",
         "score": 1,
+        "lifespan": -1
         }
 
     DRAGON_FRUIT = {
         "name": "DRAGON_FRUIT",
         "color": (145, 67, 114),
         "image": "snakes_battle\\images\\fruits\\dragonfruit.png",
         "score": 2,
+        "lifespan": -1
     }
     BOMB = {
         "name": "BOMB",
         "color": (114, 114, 114),
         "image": "snakes_battle\\images\\fruits\\bomb.png",
-        "score": -2
+        "score": -2,
+        "creation_probability": 1/100, # Will create a bomb roughly every 20 frames.
+        "lifespan": 80, # How many frames will the bomb be on the board
+    }
+    SHIELD = {
+        "name": "SHIELD",
+        "color": (68, 79, 231),
+        "image": "snakes_battle\\images\\fruits\\shield.png",
+        "creation_probability": 1/500, # Will create a bomb roughly every 20 frames.
```

Comment:
Not the correct name in the documentation, and a very repetitive one.
Just document what each item is on the top of the class instead of duplicating text across the code.